frontend/src/Editor/QueryManager/QueryManager.jsx

```diff
@@ -60,6 +60,7 @@ let QueryManager = class QueryManager extends React.Component {
         appId: props.appId,
         dataSources: props.dataSources,
         dataQueries: dataQueries,
+        appDefinition: props.appDefinition,
```

Comment:
I don't see the `appDefinition` used anywhere in the file. Can you please remove it if it is not needed?